### ISSUE TITLE
Re-linking, possible url and referer handling in images table

### ISSUE THREAD

**Issue by yaminorito** [calvinaquino/LNReader-Android#254]
After migrating an ancient db(very old and around 6 months not updated) from an old brick phone to a new tablet, big images won't load from storage(doesn't matter where it is saved), re-linking it would just fix the thumb images but not the images. After fiddling with the pages.db and logcat, I've discovered that url and referer aren't properly updated.

my old db
url: http://www.baka..... or https://www.baka..... 
referer: http://www.baka..... or https://www.baka..... 

current BT header + domain name used by the app:
url: https://baka....
referer: https://baka....

The stored images are properly linked to, but due to incorrect url and referer, the new stored image path isn't saved into the filepath. It may be needed to update those two columns during re-linking but that might need a "header" + "domainname" variable(that is updated regularly and is used by the app on most it's link functions) somewhere it would copy from whenever it re-links.

**Comment by Nandaka** [comment]
have you tried to change the referer/url using sql editor?

**Comment by yaminorito** [comment]
Yes that's what I did, it's working now. Though in the long run, it might be a good idea to optimize the filepath, url and referer handling, since the three columns' values are almost the same most of the time.

**Comment by Nandaka** [comment]
I think I can hardcode the referer (and maybe the file url, if hosted in bakatsuki) to always use current baka-tsuki url.
https://github.com/calvinaquino/LNReader-Android/blob/master/app/src/main/java/com/erakk/lnreader/Constants.java#L9

for filepath, unless the user also moved the old images, I think I can remove it from db (so it always use the current path stored in config).

**Comment by yaminorito** [comment]
Removing it can be done later, maybe we can use a 

try { 
 test(); 
} 
 catch (EpicFailException ex) 
{
 print(ex);
 oldmethod(); 

}

where test() ignores filepath and referer and use only url. This can probably be done through either image loading, re-linking, or image downloading. I haven't seen through the source code on those processes yet.
